src/components/ProductCategories.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { FaArrowAltCircleLeft, FaArrowAltCircleRight } from "react-icons/fa";
+import productCategories from "./product-categories.json";
+
+export default function ProductCategories() {
+  const [currImage, setCurrImage] = useState(0);
+
+  const prevSlide = () => {
+    setCurrImage(
+      currImage === 0 ? productCategories?.results.length - 1 : currImage - 1
+    );
+  };
+
+  const nextSlide = () => {
+    setCurrImage(
+      currImage === productCategories?.results.length - 1 ? 0 : currImage + 1
+    );
+  };
+
+  return (
+    <section className="slider">
+      <FaArrowAltCircleLeft className="left-arrow" onClick={prevSlide} />
```

Comment:
I see this is logic and components (prev and next) are shared with the FeaturedBanners component, maybe you could move this to a separate, shared component